resources/dicts/thoughts/alive/kitten.json

```diff
@@ -624,6 +647,18 @@
             "living"
         ]
     },
+    {
+        "id": "noisy_kit",
+        "thoughts": [
+            "Asked an elder for a story but keeps interrupting them"
```

Comment:
i think it would be nice to actually use r_c's name and pronouns here

Reply:
Fixed